bears/csv/CSVLintBear.py

```diff
@@ -9,6 +9,12 @@
 class CSVLintBear:  # pragma nt: no cover
     """
     Verifies using ``csvlint`` if ``.csv`` files are valid CSV or not.
+    The ``CSVLintBear`` does validation that checks:
+    - structural formatting of a CSV file
+    - delimiter-separated values (DSV) file accessible via URL, File,
+    or an IO-style object (e.g. StringIO)
```

Comment:
need alignment of sentences in one column.

Reply:
@KVGarg I can't understand you properly. Are you talking about the misalignment of line no. 15?

Reply:
Yes, need a tab space before “or”

Reply:
It was like that early on but coala or some build showed an error because of it.

Reply:
Ok 👍🏻